packages/x-date-pickers-pro/src/DateRangePickerDay/DateRangePickerDay.tsx

```diff
@@ -223,6 +259,9 @@ const DateRangePickerDayDay = styled(PickersDay, {
   ...(!ownerState.selected &&
     ownerState.isHighlighting && {
       color: theme.palette.getContrastText(alpha(theme.palette.primary.light, 0.6)),
+      [`&.${pickersDayClasses.dayOutsideMonth}`]: {
+        color: alpha(theme.palette.getContrastText(theme.palette.primary.light), 0.6),
```

Comment:
Nitpick, but IMHO this could use a bit more alpha to be clearly visible. 🤔 
To me, 0.4 produced a very nice result. 👍